docroot/themes/humsci/humsci_basic/src/scss/tools/_functions.animation.scss

```diff
@@ -1,3 +1,26 @@
+// Returns the proper transition based on the theme.
+// Filters through $hb-animation-list array to find the relevant
+// duration and timing values for the theme.
 @function hb-transition($property) {
-  @return $property $hb-animation-duration $hb-animation-timing-function;
+  $is-valid-theme: false;
+
+  @each $theme in $hb-global-theme-list {
+    @if ($theme == $hb-current-theme) {
+      $is-valid-theme: true;
+    }
+  }
+
+  @if ($is-valid-theme) {
+    $theme-animation-list: map-get($hb-animation-list, $hb-current-theme);
+    $hb-animation: $theme-animation-list[];
+
+    @if ($theme-animation-list) {
```

Comment:
actually maybe you can check: `@if (length($theme-animation-list) > 0)` https://sass-lang.com/documentation/modules/list#length

Reply:
Oh I like this! Thank you!!